_integration-schemas/selligent/owner.md

```diff
@@ -0,0 +1,38 @@
+---
+tap: "selligent"
+version: 1.0
+key: ""
+
+name: "owner"
+doc-link: 
+singer-schema: https://github.com/singer-io/tap-selligent/blob/master/tap_selligent/schemas.py
+description: "This table contains owner data."
+
+replication-method: "Full Table"
+
+replication-key:
+  name: ""
+
+api-method:
+  name: ""
+  doc-link: ""
+
+attributes:
+  - name: "asset_id"
+    type: "integer"
+    primary-key: true
+    description: ""
+    foreign-key-id: "owner-id"
```

Comment:
Just like with Klaviyo, let's comment this out for now since it doesn't point to anything.